tasks/apt.yml

```diff
@@ -1,5 +1,29 @@
 ---
 
+- name: Update apt cache
+  apt: update_cache=true cache_valid_time=600
+  when: ansible_os_family == 'Debian'
+
+- name: Install gpg agent
+  apt:
+    name: gpg-agent
+    update_cache: true
```

Comment:
this and the other following are redundant with the task above, I think these can be removed

Reply:
This one is required, without this, the tests are failing.

Reply:
I'd do it in a dedicated task as we do in ansible-openwisp2:
https://github.com/openwisp/ansible-openwisp2/blob/master/tasks/apt.yml#L3-L10

And remove the update_cache:true from each subsequent task (because it actually tries to update the cache again if cache time is not specified).

Please also add the same retry mechanism we have in ansible-openwisp2 to avoid failures for temporary network issues.